contracts/tests/constants.ts

```diff
@@ -2,7 +2,7 @@ import { MaxValues, TreeDepths } from "maci-core";
 import { G1Point, G2Point } from "maci-crypto";
 import { VerifyingKey } from "maci-domainobjs";
 
-export const duration = 15;
+export const duration = 20;
```

Comment:
Why is this updated?

Reply:
deployment time now is longer, given we have to deploy tally and mp from the maci contract.